aspnetcore/blazor/hosting-models.md

```diff
@@ -111,20 +119,31 @@ For more information on Microsoft native client frameworks, see the following re
 
 ## Which Blazor hosting model should I choose?
 
-Select the Blazor hosting model based on the app's feature requirements. The following table shows the primary considerations for selecting the hosting model.
+Select the Blazor hosting model based on the app's feature requirements. The following table shows the primary considerations for selecting the hosting model. 
+
+Technically, a Blazor Progressive Web Application (PWA) is a form of Blazor WebAssembly and not a separate hosting model. However, Blazor PWA offers a unique combination of features, so PWA appears in the following table in its own column.
```

Comment:
I'm not in favor of adding PWA to this table. This table is for comparing Blazor hosting models, and a PWA is not a Blazor hosting model.